iso12233/brick/iso12233/demos/analyzePatch.cc

```diff
@@ -55,8 +55,14 @@ int main(int argc, char* argv[])
 
   // Get file input.
   std::string comment;
+
+#if HAVE_LIBPNG
   bcv::Image<bcv::GRAY8> inputImage = bcv::readPNG<bcv::GRAY8>(
     appConfig.inputFileName, comment);
+#else
+  bcv::Image<bcv::GRAY8> inputImage = bcv::readPGM8(
+    appConfig.inputFileName, comment);
+#endif
```

Comment:
Could we also update the argument description in "parseArguments()," (line 153) below, to let the user know that if libpng/libpng++ isn't available, the application will instead try to read a .pgm?

Reply:
Not to worry... I'll resolve on this end.